SemDiff.Core/GitHub.cs

```diff
@@ -173,6 +218,23 @@ private static string ParseNextLink(IEnumerable<string> links)
             {
                 return null;
             }
+            if (CurrentSaved != null)
+            {
+                var removePRs = CurrentSaved;
+                foreach (var pr in pullRequests)
+                {
+                    foreach (var prRemove in removePRs)
+                    {
+                        if (pr.Number == prRemove.Number)
+                        {
+                            removePRs.Remove(prRemove);
```

Comment:
I am surprised this works, usually an error is thrown if an enumeration is changed while we are iterating over an enumerable. In this case items are removed from `removePRs`. I wonder if any of our test cases cover this line?